codalab/client/json_api_client.py

```diff
@@ -621,7 +621,7 @@ def fetch_contents_blob(self, target, range_=None, head=None, tail=None, truncat
         headers = {'Accept-Encoding': 'gzip'}
         if range_ is not None:
             headers['Range'] = 'bytes=%d-%d' % range_
-        params = {}
+        params = {'support_redirect': 1}
```

Comment:
nit: should `support_redirect` be conditionally set here based on other things like `head` and `tail`? 

I'm fine with it being done in `bundles.py` as well.

Reply:
I'd rather do it in `bundles.py`.